### DISCUSSION TITLE
Won't recognize Hammer SDK files

### DISCUSSION THREAD

**Discussion by ghost** [WilliamVenner/gmpublisher#162] [Help]
I'm new to everything regarding modding, so i'm probably stupid.
I used gmpublisher a few months ago and successfuly uploaded a map i made in hammer, but now i can't seem to do the same process.  I have the latest version. Is there something i'm missing?
I didn't do anything to the save files (.bsp, .lin, .log, .vmf, .vmx), but when it comes to dragging and dropping the folder with the hammer files on the upload addon window nothing shows up, it won't recognize the formats i guess. I'm almost sure i'm missing something, i remember a thing about converting file formats or something but i don't recall having to get to that point with gmpublisher, it was very easy.

Help a newbie out?

**Comment by WilliamVenner** [comment]
Try browsing for the folder instead of dragging and dropping
